### ISSUE TITLE
Provide link to instructions on comment formatting options.

### ISSUE THREAD

**Issue by myfirstnameispaul** [OdyseeTeam/odysee-frontend#1422]
**Is your feature request related to a problem? Please describe.**
I cannot find a list of format options for comments, and I can see users doing things like embedding videos, but cannot find where formatting to do this is explained.

**Describe the solution you'd like**
A list of format options for comments and how to use those options.

**Comment by tzarebczan** [comment]
Chat uses markdown, but it's more of an undocumented cool kids feature. once we get a better post/text editor, we'll make it clearer then.

**Comment by tzarebczan** [comment]
Seen this come up as a question a few times. We should consider making the chat boxes more like the post preview, ability to upload images, hints about markdown options, etc. This option should probably require premium so it's not abused.

**Comment by infinite-persistence** [comment]
Would re-enabling the `Advanced Editor` button on the upper-right be the quick fix?
That would contain the "Markdown Guide" in the toolbar, and upload-image-hint at the bottom.

**Comment by tzarebczan** [comment]
Yeah, worth a shot. A way to turn on / off like reddit could be neat (maybe it settings to start). Preview can be wonky at times.

Can't wait for a better editor at some point 🤣
